### ISSUE TITLE
Skeleton's speed is ridiculously bad.

### ISSUE THREAD

**Issue by Megamaw** [d3athrow/vgstation13#12418]
I somehow don't think a skeleton should be as slow as a Diona. I can understand Diona being slow, regenerating health quickly and never needing to eat.

Skeletons die. Fast. Two toolboxes to the torso and they're in crit. For a race that dies in a few hits, I'd at least imagine them, from a balancing perspective, to be relatively fast so they can at least dodge.

Nope. Slow as molasses. And unlike Diona, they still need to eat for some reason? But for some reason, they can only eat raw meat? So, unless the chef decides to keep skeletons in particular in mind, as the round goes on you'll be even slower.

Seriously. Skeletons shouldn't be that slow.

**Comment by TheFriggenSawbones** [comment]
I really think some of the nerfs Skeletons recieved before/during their removal should be reverted.

So they have,

Slow Movespeed
Forced to eat Raw Meat (can't drink milk)
Can't take pills/they're a medbay nightmare.
2x brute damage taken.

but they don't need to breathe and they don't bleed.
It doesn't feel like a good tradeoff

**Comment by Gringonius** [comment]
Live by the meme, die by the meme.

**Comment by clusterfack** [comment]
Remove skeletons again

**Comment by AutismSpider** [comment]
can eye drop/rub them down with a rag to get chems in them.

so rub that skeleton down to heal him/her

**Comment by Intigracy** [comment]
Remove skeletons again

**Comment by ihadtoregisterforthis** [comment]
Up the speed, add milk to allowed drinks and guarantee rags on all stations.

**Comment by gbasood** [comment]
@TheFriggenSawbones drinking milk is feature loss not an intentional nerf
the rest is pretty much standard for nonhumans

**Comment by Ephx1** [comment]
It's a step above cluwne, another walking joke, sounds fine to me

**Comment by Exxion** [comment]
@gbasood 

> standard for nonhumans

Dude
Absolutely none of those are shared with any other nonhuman except slow movement with dionae

Wait
Plasmamen take increased brute too now, don't they?
But not increased by THAT much

**Comment by gbasood** [comment]
@Exxion Dionae also take increased burn damage which is MORE THAN ENTIRELY OFFSET by the fact that brute might as well not exist for them
but otherwise they share that too
I thought Vox were slower off the top of my head but then I realized I'm dumb
Plasmen are slower, aren't they? Or no?

By  the way I don't think balancing a gimmick race to be easier to play or even on par with standard humans is not a serious balance issue. Doesn't mean we shouldn't but fuck why do skeleton issues always get the most activity

It's like there's a very vocal minority who would prefer their station be nothing but memes

**Comment by SonixApache** [comment]
fuck skeletons

**Comment by Exxion** [comment]
> Dionae also take increased burn damage which is MORE THAN ENTIRELY OFFSET

Honestly Dionae are probably OVERALL the most balanced race aside from humans and vox (which are basically identical)
In specific situations they can be either impossibly robust or impossibly useless, but they generally even out
(Example: A diona can kill any number of goliaths at once with no armor and only a melee weapon, given enough time and light. But a fire trap will kill them in one hit, and most lasers will crit them in 1-2.)

Wait this issue is about skeletons
Yeah skeletons sound like they were nerfed too hard

**Comment by Megamaw** [comment]
one time, as a diona while cuffed, in a small room in maintenance, i was trapped with a ling actively trying to kill me in the same room.

i lived for five minutes.

either shit's really strong or i'm just used to playing a weak race.

@gbasood afaik i'm the only one bringing up these skeleton issues on github. i'm not asking for skeletons to be broken, nor am i asking for them to even be on par with the other super-powerful races like diona, plasmaman, and human. i'm just asking for them to have SOMETHING going for them.

**Comment by Shadowmech88** [comment]
You know, you could make a PR to buff skeleton speed yourself.

All it takes is changing literally one line.

**Comment by AutismSpider** [comment]
Plasma men and skeletons both take same level of brute damage. Plasma men are only slow with the suit on with it off they move at normal human speed same with Vox suit slows them down.

**Comment by Shadowmech88** [comment]
@AutismSpider 
Plasmamen have a brute_mod of 1.5, skeletons have a brute_mod of 2.0.

**Comment by AutismSpider** [comment]
Sure its 1.5 for plasmamen? Been ZAS slam into walls and put into near crit as plasma man

**Comment by Shadowmech88** [comment]
I just looked at the code, and it is.

https://github.com/d3athrow/vgstation13/blob/Bleeding-Edge/code/modules/mob/living/carbon/human/plasmaman/species.dm#L22

**Comment by Exxion** [comment]
You can get ZAS slammed into a wall and put into crit as ANY race
You can probably get insta_killed_ by a hard enough ZAS slam

**Comment by ghost** [comment]
Jesus christ megamaw you are a one-issue guy, aren't you?

**Comment by Intigracy** [comment]
Skeletons have being a meme going for them.

**Comment by gbasood** [comment]
I'd honestly prefer skeletons stay the way they are so they can be hardmode race
The only more hardmode you could get is playing a catbeast by choice, honestly

**Comment by Megamaw** [comment]
@gbasood hardmode is one thing, but being one of the slowest races and dying in two toolboxes is another.

**Comment by The-Albinobigfoot** [comment]
> hardmode

**Comment by gbasood** [comment]
Let me rephrase
The brute multiplier should at least be reduced, say to what dionae's multiplier for burn is
I don't know enough _from experience_ to say what I feel about skele speed

Feature loss things like no milk is still bad

**Comment by Exxion** [comment]
Dude
Diona burn mod is 2.5, if memory serves

**Comment by Shadowmech88** [comment]
Just checked, it is 2.5.

**Comment by gbasood** [comment]
ffffff
1.5 brute mod is what I meant

**Comment by ComicIronic** [comment]
I cannot believe we are continuing to have issues about this meme species.

**Comment by ghost** [comment]
Playing your shitty meme race is a privilege you don't deserve but now you're afforded it you should accept the fact that an animated skeleton is unlikely to be as strong as a human.

**Comment by NigglyWiggly** [comment]
Celt hates skeletons almost as much as everyone else hates catbeasts and shit.

**Comment by gbasood** [comment]
I actually hate skeletons more than catbeasts
At least catbeasts are valid and die within 5 minutes, and are non-playable by default

**Comment by Intigracy** [comment]
The obvious solution is to remove skeletons.
